lib/rules/use-ember-get-and-set.js

```diff
@@ -13,33 +17,80 @@ module.exports = {
       category: 'General',
       recommended: true
     },
-    fixable: null, // or "code" or "whitespace"
+    fixable: 'code',
   },
 
   create(context) {
+    const sourceCode = context.getSourceCode();
+    const filename = context.getFilename();
     const message = 'Use get/set';
 
-    const report = function (node) {
-      context.report(node, message);
-    };
+    function isFileInDirectory(dirname) {
+      const pathParts = filename.split(path.sep);
+      return pathParts.indexOf(dirname) > -1;
+    }
+
+    function report(node) {
+      context.report({
+        node: node.callee.property,
+        message,
+        fix(fixer) {
+          // this.set('foo', 3);
+          // └┬─┘ └┬┘ └───┬──┘
+          //  │    │      └─────── args
+          //  │    └────────────── method
+          //  └─────────────────── subject
+
+          const subject = sourceCode.getText(node.callee.object);
+          const method = sourceCode.getText(node.callee.property);
+          const args = node.arguments.map(a => sourceCode.getText(a)).join(', ');
+
+          const fixedSource = `${method}(${subject}, ${args})`;
 
-    const avoidedProperties = [
+          return fixer.replaceText(node, fixedSource);
+        }
+      });
+    }
+
+    const avoidedMethods = [
       'get',
       'set',
       'getProperties',
       'setProperties',
       'getWithDefault',
     ];
 
+    const testMethodsToSkip = [
+      'get',
+      'set',
+    ];
+
+    const directoriesToSkipCompletely = [
```

Comment:
Right, a `.eslintrc` file does seem like the easiest approach here. So I think we can leave out the option for now and rely on users excluding directories on their own, I like this :)